site/installing.md

```diff
@@ -177,27 +177,21 @@ After Scope has been launched, open your web browser to [https://cloud.weave.wor
 
 ##<a name="k8s"></a>Installing Scope on Kubernetes
 
-To install Scope onto a Kubernetes cluster in Standalone Mode, follow these instructions.
+### With Weave Cloud (recommended)
 
-**Before You Begin**
+Weave Cloud runs the Scope UI for you, so you don't have to configure exposing and securing it for your team.
```

Comment:
plus keeping it running. Though the whole "so you don't have to configure exposing and securing it" reads awkwardly already.